azurelinuxagent/ga/monitor.py

```diff
@@ -167,8 +167,8 @@ def init_sysinfo(self):
                                                  DISTRO_CODE_NAME,
                                                  platform.release())
         self.sysinfo.append(TelemetryEventParam("OSVersion", osversion))
-        self.sysinfo.append(
-            TelemetryEventParam("GAVersion", CURRENT_AGENT))
+        self.sysinfo.append(TelemetryEventParam("GAVersion", CURRENT_AGENT))
```

Comment:
A note about telemetry improvement: the agent version should be set when the event is being created and populated, not on agent startup when this method is called, since the version can change in the meantime.
I think this is more related to the task of sending the actual timestamp than the scope of this PR, though.

Reply:
With the changes in #1663, this can be added easily in the `_add_default_parameters_to_event` method added in the `EventLogger` class. Once that goes in, this change would be trivial.